### ISSUE TITLE
Unable to determine project id.

### ISSUE THREAD

**Issue by s2t2** [googleapis/google-api-ruby-client#750]
Hi, I'm seeing "Unable to determine project id." warning messages when making API calls with the calendar service. How can I suppress/fix these warning messages? Thanks!

I think this is [the source](https://github.com/googleapis/google-auth-library-ruby/blob/0852eb9ced41d4597807bee787b36a56fbe2d9d2/lib/googleauth/credentials_loader.rb#L142-L150).

#### Environment details

  - OS: Mac
  - Ruby version: 2.5.3
  - Gem name and version: google-api-client (0.27.0)

from gemfile.lock:

```
  google-api-client (0.27.0)
      addressable (~> 2.5, >= 2.5.1)
      googleauth (>= 0.5, < 0.8.0)
      httpclient (>= 2.8.1, < 3.0)
      mime-types (~> 3.0)
      representable (~> 3.0)
      retriable (>= 2.0, < 4.0)
      signet (~> 0.10)
   googleauth (0.7.1)
      faraday (~> 0.12)
      jwt (>= 1.4, < 3.0)
      memoist (~> 0.16)
      multi_json (~> 1.11)
      os (>= 0.9, < 2.0)
      signet (~> 0.7)
```

#### Steps to reproduce

  1. make an API call using the calendar service (`Google::Apis::CalendarV3::CalendarService`)

#### Code example

Given a client which is an instance of `Google::Apis::CalendarV3::CalendarService`:

```ruby
client.list_calendar_lists
```

**Comment by blowmage** [comment]
Can you post the error backtrace?

**Comment by s2t2** [comment]
I'm not seeing an error with a backtrace, just a string warning message. 

In my CI logs, I'm seeing some [references](https://travis-ci.com/prof-rossetti/my-banner-rb/builds/95202270#L609-L620) to the gcloud cli. I don't remember seeing these before, but I'll be looking into them.

**Comment by blowmage** [comment]
This was resolved in googleauth 0.7.2.
